src/app/patient-table/patient-table.component.html

```diff
@@ -1,6 +1,10 @@
 <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
     <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-    <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+    <tr
+        mat-row
+        *matRowDef="let row; columns: displayedColumns"
+        (click)="sendVisit(row)"
```

Comment:
`sendVisit` not the best name for a function, maybe: `startNewVisit` or `startNewAppointment` or `newAppointment`

Reply:
done